packages/core/src/components/DataTable.story.tsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import DataTable from './DataTable';
+import { SelectedRows, TableRow } from './DataTable/types';
+
+// @ts-ignore
```

Comment:
Do not ignore these, just use the alias imports. `:storybook/components/DataTable/DataTableData`